src/routes/Review.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react"
+import { Link, useParams } from "react-router-dom"
+import * as api from "../api"
+
+export default function Review() {
+	const { review_id } = useParams()
+
+	const [reviewData, setReviewData] = useState()
+	const [loading, setLoading] = useState(true)
+
+	useEffect(() => {
+		setLoading(true)
+		api.fetchReview(review_id).then(({ review }) => {
+			setReviewData(review)
+			setLoading(false)
+		})
+	}, [])
```

Comment:
the page is refreshing for me every 10 seconds (could be because I'm running it in a sandbox and it might not like vite) but try putting the review id in the dependency array and see does that sort it